frontend/helpers/revisions.ts

```diff
@@ -0,0 +1,11 @@
+import { get } from 'lodash';
+import memoizeOne from 'memoize-one';
+
+export const getActiveRevisions = memoizeOne((revisions) => {
+	return revisions.filter((revision) => !revision.void);
+});
+
+export const getCurrentRevisionId = memoizeOne((revisions) => {
+	const activeRevisions = getActiveRevisions(revisions);
+	return get(activeRevisions[0], '_id');
+});
```

Comment:
Yes. It's because of that revisions dialog is shared between Teamspaces and Viewer, so we are getting revisions data in two different ways.
In case of Teamspaces we have a information about current revision. We get it from url params.
In case of Viewer, we don't have that information if "active" revision is opened.

Nevertheless I refactored the code a bit. :)